### PR TITLE
feat: tooltip: add sizes to tooltip

### PR DESCRIPTION
**Pull request by ychhabra-eightfold**
## SUMMARY:
feat: tooltip: add sizes to tooltip

## CHANGE TYPE:

-   [ ] Bugfix Pull Request
-   [ ] Feature Pull Request

## TEST COVERAGE:

-   [ ] Tests for this change already exist
-   [ ] I have added unittests for this change

### CODE REVIEW

File: src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React, { FC, useEffect, useRef, useState } from 'react';
 import {
-    useFloating,
-    shift,
-    autoUpdate,
     arrow,
+    autoUpdate,
     offset as fOffset,
+    shift,
+    useFloating,
```
Comment (jhoward-eightfold) [inline src/components/Tooltip/Tooltip.tsx:7]: If we want to all get on the same page with alphasorting, etc, there is a vscode plugin called TypeScript Hero which will auto sort import statements.

I believe you were opposed to adding a .vscode configuration file that would recommend extensions? Lets revisit that conversation soon. Its a relatively common thing for open source projects and can make onboarding new contributors easier.

File: src/components/Tooltip/tooltip.module.scss [hunk 2]
```diff
@@ -74,4 +73,16 @@ $tooltip-arrow-shadow: 1px 1px 2px rgba(15, 20, 31, 0.12);
...
+        max-width: 140px;
+    }
+
+    &.medium {
+        max-width: 240px;
+    }
+
+    &.large {
+        max-width: 360px;
+    }
```
Comment (jhoward-eightfold) [inline src/components/Tooltip/tooltip.module.scss:87]: is there a default, or it will be allowed to grow as large as necessary?

**Review by jhoward-eightfold — APPROVED** [review]

File: src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (ychhabra-eightfold) [reply src/components/Tooltip/Tooltip.tsx:7]: sure we can discuss that, but i dont use vscode 😄

File: src/components/Tooltip/tooltip.module.scss (hunk 2 above)
Comment (ychhabra-eightfold) [reply src/components/Tooltip/tooltip.module.scss:87]: good catch default should be small, making that change

**Pull request merged**